platform/rct/Makefile

```diff
@@ -14,6 +14,13 @@ LD := $(CROSS_COMPILE)g++
 OBJDUMP := $(CROSS_COMPILE)objdump
 OBJCOPY := $(CROSS_COMPILE)objcopy
 
+ASL_FW_POSTPROCESS := asl_f4_fw_postprocess
+ifeq ($(PYTHON),python2.7)
```

Comment:
Kill it then.  asl_f4_fw_postprocess should be in the path or it should raise an error.